stdlib/public/core/VarArgs.swift

```diff
@@ -469,30 +469,30 @@ final internal class _VaListBuilder {
                Builtin.addressof(&self.header)))
   }
 
-  @usableFromInline // FIXME(sil-serialize-all)
+  @usableFromInline // c-abi
   internal var gpRegistersUsed = 0
-  @usableFromInline // FIXME(sil-serialize-all)
+  @usableFromInline // c-abi
   internal var fpRegistersUsed = 0
 
-  @usableFromInline // FIXME(sil-serialize-all)
+  @usableFromInline // c-abi
   final  // Property must be final since it is used by Builtin.addressof.
   internal var header = Header()
-  @usableFromInline // FIXME(sil-serialize-all)
+  @usableFromInline // c-abi
   internal var storage: ContiguousArray<Int>
 }
 
 #else
 
 /// An object that can manage the lifetime of storage backing a
 /// `CVaListPointer`.
-@_fixed_layout // FIXME(sil-serialize-all)
-@usableFromInline // FIXME(sil-serialize-all)
+@_fixed_layout // c-abi
```

Comment:
Sorry, the *result* is unlikely to change, and the implementation of the various methods is unlikely to be incorrect, but that doesn't *necessarily* mean that we won't need *more* fields to support *additions* to the ABI. So `inlinable` is probably fine, but `fixed_layout` might not be.

…except I also said (in chat) that we want this to compile all the way down to what C does. If we're not actually doing that, though, maybe it's not worth it.

Reply:
I feel like if you go for full inlinability it's a bit weird to not mark it fixed layout. The scope of changes that leaves on a type like this is pretty minimal.

Reply:
Having looked at it and talked to John about the vararg ABI, seems pretty unlikely we'd ever need to add anything else to the class, and not keeping it fixed layout is a pain. Since it's an internal helper class, the cleanest way if we did end up revamping this would probably be to just create a different class and use that in later revisions, keeping the old one (with its old logic) around.